content/hi/docs/reference/glossary/cluster-architect.md

```diff
@@ -0,0 +1,17 @@
+---
+title: क्लस्टर आर्किटेक्ट (Cluster Architect)

```

Comment:
For this one, no need to include the original English term (IMO)
```suggestion
title: क्लस्टर आर्किटेक्ट
```

Reply:
+1 for this